aws-py-apigatewayv2-eventbridge/README.md

```diff
@@ -0,0 +1,71 @@
+# API Gateway to EventBridge
+
+[![Deploy with Pulumi](https://get.pulumi.com/new/button.svg)]([![Deploy with Pulumi](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new?template=https://github.com/pulumi/examples/aws-py-apigatewayv2-eventbridge))
+
+This example creates an AWS API Gateway proxy integration with EventBridge and Lambda. It defines a single API Gateway endpoint that publishes events to an EventBridge event bus, and an accompanying event rule that matches those events and invokes a Lambda function.
+
+![Diagram showing an API Gateway-EventBridge integration](https://user-images.githubusercontent.com/274700/149042008-ce1bedd3-f795-409f-b8ab-57d1ff78c259.png)
+
+## Prerequisites
+
+1. [Install Pulumi](https://www.pulumi.com/docs/get-started/install/).
+1. [Install Python](https://www.pulumi.com/docs/intro/languages/python/).
+1. Configure your [AWS credentials](https://www.pulumi.com/docs/intro/cloud-providers/aws/setup/).
+
+### Deploying the App
+
+1. Clone this repo, change to this directory, then create a new [stack](https://www.pulumi.com/docs/intro/concepts/stack/) for the project:
+
+    ```bash
+    pulumi stack init
+    ```
+
+1. Specify an AWS region to deploy into:
+
+    ```bash
+    pulumi config set aws:region us-west-2
+    ```
+
+1. Install Python dependencies and run Pulumi:
+
+    ```bash
+    python3 -m venv venv
+    venv/bin/pip install -r requirements.txt
```

Comment:
It's better to activate the venv before running anything. Also, note that pip will automatically be pip3 when you use python3 to create the venv; there's also other flags to ensure you're using Python 3 if you're working on a different setup than the base one, but I doubt someone who has that all set up will be going by these directions verbatim 😉  
```suggestion
    python3 -m venv venv
    source venv/bin/activate
    pip install -r requirements.txt
```